src/utils/mutate.js

```diff
@@ -0,0 +1,242 @@
+import { chunk, cloneDeep, flatten, isFunction, mapValues } from 'lodash';
+import debug from 'debug';
+import { firestoreRef } from './query';
+import mark from './profiling';
+
+const info = debug('rrf:mutate');
+
+/**
+ * @param {object} firestore
+ * @param {string} collection
+ * @param {string} doc
+ * @returns Boolean
+ */
+const docRef = (firestore, collection, doc) =>
```

Comment:
Wanted to call out that here we could use a slash path instead of making a util function, this also handles many levels deep:

```js
firestore.doc('collection1/doc1/subcollection/subcollectDoc')
```

Reply:
fixed!